internal/pkg/dependencies/abstract.go

```diff
@@ -0,0 +1,54 @@
+package dependencies
```

Comment:
Vytvoril som novy PKG `dependencies`, doteraz sme mali iba CLI dependencies.
Potrebujem lepsie riesit zavislosti v testoch + pouzijeme to aj ked budeme implementovat webservice.
Kedze niektore zavislosti su pre CLI/webservice rovnake, a niektore sa budu riesit rozne.